tarantool/schema.py

```diff
@@ -54,14 +56,24 @@ def __init__(self, space_row, schema):
         self.arity = space_row[1]
         self.name = space_row[2]
         if isinstance(self.name, bytes):
-            self.name = self.name.decode()
+            self.name = self.name.decode(encoding='utf-8')
```

Comment:
nit: Is it reasonable to set global variable for keeping the encoding value? You have eight the same part of the code.

Reply:
It is about PR #172, but you're right. I'll create necessary wrappers.

Reply:
I rewrote this part, see PR #172.